src/Mono.Android/Android.Runtime/TypeManager.cs

```diff
@@ -13,9 +13,9 @@ public static string LookupTypeMapping (string[] mappings, string javaType)
 			return Java.Interop.TypeManager.LookupTypeMapping (mappings, javaType);
 		}
 
-		public static void RegisterType (string java_class, Type t)
+		public static void RegisterType (string java_class, Type t, bool ignoreExistingRegistration = false)
```

Comment:
This what `JNIEnv` calls hence the change that needs to be propagated. I can add an overload instead to keep the old member if that's used externally.